app/Http/Kernel.php

```diff
@@ -39,6 +39,7 @@ class Kernel extends HttpKernel
      */
     protected $routeMiddleware = [
         'auth' => \App\Http\Middleware\Authenticate::class,
+        \Illuminate\Routing\Middleware\ThrottleRequests::class,
```

Comment:
Looks like the `'throttle'` key is missing 👀 @jasonmccreary

Reply:
That's not exactly it, it actually [belongs in the priority group](https://github.com/laravel/laravel/blob/master/app/Http/Kernel.php#L76). That's where it should have been added. I'll tweak the hooks to ensure it skips this part.